src/components/FavoriteBuildings.js

```diff
@@ -0,0 +1,51 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { patchUserData } from '../store/userSlice';
+
+import useAxios from '../hooks/useAxios';
+
+export default function FavoriteBuildings() {
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+  const user = useSelector((state) => state.user);
```

Comment:
```suggestion
  const userInformation = useSelector((state) => state.user.userInformation);
```

제가 읽었던 자료가 지금 출처를 다시 찾아봐야되긴 하는데 user 이렇게 한번에 부르는 것은 꼭 필요한 경우가 아니면 지양하라고 들었습니다. 이렇게 쓰면 다른 컴포넌트에서 user 내부에 useInformation이 아닌 다른 state를 변경하면 여기서 리렌더링이 될 필요가 없음에도(userInformation만 쓰니까) 리렌더링이 되버린다고 하네요.

Reply:
아 바로 바꾸겠습니다. 
처음 알았네요